app/views/expenses/_form.html.erb

```diff
@@ -1,26 +1,26 @@
-<div class="expense-form">
+<%= render partial: 'shared/nav', locals: {title:'TRANSACTION', symbol:'←', url: user_groups_path}%>
```

Comment:
### "Transactions page"

- [ ] The same thing for the Transactions page, When the user clicks on the "Back" button (<), the user should always navigates to the home page or Category page 👍🏻